Gemfile

```diff
@@ -10,3 +10,11 @@ group :coverage, optional: ENV['COVERAGE']!='yes' do
   gem 'simplecov-console', :require => false
   gem 'codecov', :require => false
 end
+
+group :development do
+  gem 'rubocop', '~> 1.6.1'
+  gem 'rubocop-rspec', '~> 2.0.1'
+  gem 'rubocop-performance', '~> 1.9.1'
+
+  gem 'faraday-retry', '~> 2.0.0'
```

Comment:
Shouldn't this be part of the release group? Rubocop doesn't use faraday.

Reply:
moved 👍